JotunnLib/Documentation/tutorials/utils/config.md

```diff
@@ -0,0 +1,26 @@
+﻿# Persistent & Synced Configurations
+
+Jötunn itself does not provide any implementations or abstractions for persisent configurations. We do however respect [BepInEx.ConfigEntry](https://bepinex.github.io/bepinex_docs/master/articles/dev_guide/plugin_tutorial/3_configuration.html)'s, their various properties, as well as their [ConfigurationManager](https://github.com/BepInEx/BepInEx.ConfigurationManager) properties. Furthermore we have implemented a method of enforcing server side sync on specific configs via the [ConfigurationManagerAttributes](xref:ConfigurationManagerAttributes) `IsAdminOnly` flag.
```

Comment:
For the ConfigurationManagerAttributes we should add a hint, so mod developers will use JVL's implementation of the ConfigurationManagerAttributes. The original implementation does not include the admin only flag. (so xref to JotunnLib.ConfigurationManagerAttributes)